### ISSUE TITLE
Missing layers

### ISSUE THREAD

**Issue by RonRichman** [rstudio/keras#1152]
Hi everyone

It seems at least two layers are missing from R Keras if we compare to the Keras API docs. 

LayerNorm:
https://keras.io/api/layers/normalization_layers/layer_normalization/

Multihead Attention:
https://keras.io/api/layers/attention_layers/multi_head_attention/

Can these please be added?

**Comment by dfalbel** [comment]
Thanks @RonRichman 
Both should be working in the dev version of Keras. We will submit to CRAN soon.
